### DISCUSSION TITLE
[Feature] Mobile Search Filter Additions / Select-All

### DISCUSSION THREAD

**Discussion by patienttruth** [immich-app/immich#14481] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I've just gone through the process of clearing out space on my device as it was full. It worked, but was a little clunky.

I understand #165 is on the horizon, but in the interim would it be possible to add:
- "select-all" to the image browser, e.g. under library > on this device > etc, and in search results?
- filters on search results for size and "is backed up", "device only"

I couldn't find anything related except a mention of "select-all" under #165. That feature would be useful for more than just this use case, e.g. for creating albums from search results.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#3751/#17457
